benchmarks/benchmarks-stack/benchmarks-stack/lib/canary-policy-doc.json

```diff
@@ -0,0 +1,118 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "VisualEditor0",
```

Comment:
Extra credit: rename this Sid to something more useful and sort the actions so that we can more easily see what is allowed for each service.

Reply:
yeah, changed, also removed the actions related to CFN, I think I added them when we want EC2 to delete the CFN stack. Now, we use Lambda instead